src/components/PageTitleBar/PageTitleBar.jsx

```diff
@@ -60,6 +60,12 @@ const defaultProps = {
   content: undefined,
 };
 
+// do typecheck to support old markup with surrounding 'p' tag
+// eslint-disable-next-line
```

Comment:
Scope the ignore to specific rule(s) so there's less exposure to potential errors.

I assume this is because only one export is allowed per file or something similar? You could inline the logic in the jsx and test the `p` tag is/isn't rendered appropriately by passing the node through PageTitleBar instead of directly mounting and testing `TooltipDescription`.

Reply:
I applied this because I am using the same description that is used in the component. It already has prop-type validation but eslint is expecting prop type validation again for this as well.